app/models/topic.rb

```diff
@@ -830,6 +832,23 @@ def invite_by_email(invited_by, email, group_ids = nil, custom_message = nil)
     Invite.invite_by_email(email, invited_by, self, group_ids, custom_message)
   end
 
+  def invite_existing_muted?(target_user, invited_by)
+    # check if inviter is muted
```

Comment:
In my opinion, I don't think we need the comment here since the db query would explain what we're trying to do.